low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -301,12 +302,12 @@ if __name__ == "__main__":
     if not os.path.isfile(onGoingRequestPath):
         if len(sys.argv) >= 3:
             tmpFile = open(onGoingRequestPath, "w")
-            os.makedirs(f"/tmp/{sspl_bundle_tmp}")
```

Comment:
We can also cleanup the `sspl_bundle_tmp` if it already exists. If any of below three steps fails for some reason, we may have left over files.

Reply:
Done